filters/CMakeLists.txt

```diff
@@ -8,126 +8,127 @@ PCL_SUBSYS_DEPEND(build "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS})
+  src/passthrough.cpp
+  src/shadowpoints.cpp
+  src/project_inliers.cpp
+  src/radius_outlier_removal.cpp
+  src/random_sample.cpp
+  src/normal_space.cpp
+  src/sampling_surface_normal.cpp
+  src/statistical_outlier_removal.cpp
+  src/voxel_grid.cpp
+  src/approximate_voxel_grid.cpp
+  src/bilateral.cpp
+  src/fast_bilateral.cpp
+  src/fast_bilateral_omp.cpp
+  src/crop_hull.cpp
+  src/voxel_grid_covariance.cpp
+  src/voxel_grid_label.cpp
+  src/frustum_culling.cpp
+  src/covariance_sampling.cpp
+  src/median_filter.cpp
+  src/uniform_sampling.cpp
+  src/voxel_grid_occlusion_estimation.cpp
+  src/normal_refinement.cpp
+  src/grid_minimum.cpp
+  src/morphological_filter.cpp
+  src/local_maximum.cpp
+  src/model_outlier_removal.cpp
+)
 
-    set(impl_incs
-        "include/pcl/${SUBSYS_NAME}/impl/conditional_removal.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/crop_box.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/crop_hull.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/plane_clipper3D.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/box_clipper3D.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/extract_indices.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/filter.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/filter_indices.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/passthrough.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/shadowpoints.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/project_inliers.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/radius_outlier_removal.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/random_sample.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/normal_space.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/sampling_surface_normal.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/statistical_outlier_removal.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/voxel_grid.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/approximate_voxel_grid.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/bilateral.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/fast_bilateral.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/fast_bilateral_omp.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/voxel_grid_covariance.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/convolution.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/convolution_3d.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/voxel_grid_occlusion_estimation.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/frustum_culling.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/covariance_sampling.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/median_filter.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/uniform_sampling.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/normal_refinement.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/grid_minimum.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/morphological_filter.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/local_maximum.hpp"
-        "include/pcl/${SUBSYS_NAME}/impl/model_outlier_removal.hpp"
-        )
+set(incs
+	"include/pcl/${SUBSYS_NAME}/boost.h"
```

Comment:
Tabs here as well.
Please grep for tabs, I may have missed some.

Reply:
I have replaced now all tabs in every CMake file, even they have no adjustment regarding nesting. In case you wish to squash both commits, tell me.

Reply:
:+1:
No need to do anything for you, I can squash while merging from GitHub interface.